robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -191,50 +197,42 @@ def subscriber_callback(message):
 def publish_pds_data(message):
     # parse the data received from PDS
     # converts message from string to float
-    dataPDS = message.split(',')  ###returns an array of ALL data from the PDS
+    dataPDS = message.split(',')  # returns an array of ALL data from the PDS
     # create the message to be published
     voltage = Float32()
     current = JointState()
```

Comment:
I secretly wish the current wasnt a "JOINT STATE" lmao

Reply:
Yeah, I think my initial intent was to use the JointState types for motor currents since current is directly proportional to torque so it could fit in the "effort" parameter.

Here are some proposed solutions:
- we make a custom message type
- we use the 'effort' parameter
- we inspire ourselves off of this post https://answers.ros.org/question/211635/topic-for-motor-current/

Reply:
@vashmata If i understand correctly, the currents are just a series of floats?
If so, then why not just create a message type that either
1. a single float array or
2. a series of floats each one with an identifier such as C1 C2 C3, etc.

Reply:
That works too, I just like the idea of having a custom message type for it. But if you feel like a float[6] with a well-chosen topic name is enough then that's fine too

Reply:
I think a float array would be superior. First in terms of memory, it would end up being the same thing. But if we use an array, the message might be reusable for different currents (from another module), which might have a different amount of currents to publish. So going with an array seems more flexible

Reply:
That's what I went for in the end